### PR TITLE
Verify tag

### PR DESCRIPTION
**Pull request by pangthebabydustbunny**

### CODE REVIEW

File: src/pages/AccountIndexPage.re [hunk 1]
```diff
@@ -216,6 +213,7 @@ let make = (~address, ~hashtag: Route.account_tab_t) => {
   let qrCode = () => {
     address->QRCode->OpenModal->dispatchModal;
   };
+  let addressTag = address->Address.toBech32->VerifyAccount.parseAddressName;
```
Comment (evilpeach) [inline src/pages/AccountIndexPage.re:216]: 1. Move `Address.toBech32` to `VerifiedAccount.re`
2. Change `parseAddressName` to `getNameOpt`.
3. Change VerifyAccount to VerifiedAccount.

File: src/reusable/AddressRender.re [hunk 2]
```diff
@@ -74,6 +75,9 @@ let make =
 
   let ({ThemeContext.theme}, _) = React.useContext(ThemeContext.context);
 
+  let fullAddress = prefix ++ noPrefixAddress;
```
Comment (evilpeach) [inline src/reusable/AddressRender.re:78]: It's no need to use it. Just use address from prop.

File: src/utils/VerifyAccount.re [hunk 3]
```diff
@@ -0,0 +1,4 @@
+let parseAddressName =
+  fun
```
Comment (evilpeach) [inline src/utils/VerifyAccount.re:2]: Use `Address.toBech32` here

File: src/reusable/AddressRender.re [hunk 4]
```diff
@@ -94,8 +98,19 @@ let make =
           Styles.font(position),
           wordBreak ? Styles.wordBreak : "",
         ])}>
-        <span className=Styles.prefix> {prefix |> React.string} </span>
-        {noPrefixAddress |> React.string}
+        {label
+           ? switch (addressTag) {
```
Comment (evilpeach) [inline src/reusable/AddressRender.re:102]: Use like this
```
switch(addressTagOpt, label) {
| (Some(tag), true) => <span> tag->React.string </span>
| _ => <>
                 <span className=Styles.prefix> {prefix |> React.string} </span>
                 {noPrefixAddress |> React.string}
               </>
}

```

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by pzshine — APPROVED** [review]
LGTM

File: src/reusable/AddressRender.re [hunk 5]
```diff
@@ -63,6 +63,7 @@ let make =
       ~copy=false,
       ~clickable=true,
       ~wordBreak=false,
+      ~tag=true,
```
Comment (evilpeach) [inline src/reusable/AddressRender.re:66]: I think we should use `~showName`

File: src/reusable/AddressRender.re [hunk 6]
```diff
@@ -74,6 +75,8 @@ let make =
 
   let ({ThemeContext.theme}, _) = React.useContext(ThemeContext.context);
 
+  let addressTagOpt = address->VerifiedAccount.getNameOpt;
```
Comment (evilpeach) [inline src/reusable/AddressRender.re:78]: Remove this one.

File: src/reusable/AddressRender.re [hunk 7]
```diff
@@ -94,8 +97,14 @@ let make =
           Styles.font(position),
           wordBreak ? Styles.wordBreak : "",
         ])}>
-        <span className=Styles.prefix> {prefix |> React.string} </span>
-        {noPrefixAddress |> React.string}
+        {switch (addressTagOpt, tag) {
```
Comment (evilpeach) [inline src/reusable/AddressRender.re:100]: Replace `addressTagOpt` with `address->VerifiedAccount.getNameOpt`

File: src/reusable/AddressRender.re [hunk 8]
```diff
@@ -94,8 +97,14 @@ let make =
...
+         | (Some(tag), true) => <span> tag->React.string </span>
```
Comment (evilpeach) [inline src/reusable/AddressRender.re:101]: Use `name` instead of `tag`

File: src/pages/AccountIndexPage.re [hunk 9]
```diff
@@ -216,6 +213,7 @@ let make = (~address, ~hashtag: Route.account_tab_t) => {
   let qrCode = () => {
     address->QRCode->OpenModal->dispatchModal;
   };
+  let addressTag = address->VerifiedAccount.getNameOpt;
```
Comment (evilpeach) [inline src/pages/AccountIndexPage.re:216]: Remove this one and move to switch case.

File: src/pages/AccountIndexPage.re [hunk 10]
```diff
@@ -243,9 +240,17 @@ let make = (~address, ~hashtag: Route.account_tab_t) => {
                       position=AddressRender.Subtitle
                       copy=true
                       clickable=false
+                      tag=false
                     />
                   </div>
                 </div>
+                {switch (addressTag) {
+                 | Some(v) =>
```
Comment (evilpeach) [inline src/pages/AccountIndexPage.re:248]: name

File: src/pages/AccountIndexPage.re [hunk 11]
```diff
@@ -243,9 +240,17 @@ let make = (~address, ~hashtag: Route.account_tab_t) => {
                       position=AddressRender.Subtitle
                       copy=true
                       clickable=false
+                      tag=false
                     />
                   </div>
                 </div>
+                {switch (addressTag) {
```
Comment (evilpeach) [inline src/pages/AccountIndexPage.re:247]: move get function to here.

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by evilpeach — APPROVED** [review]

**Pull request merged**